cupyx/scipy/sparse/base.py

```diff
@@ -434,9 +434,27 @@ def multiply(self, other):
     def power(self, n, dtype=None):
         return self.tocsr().power(n, dtype=dtype)
 
-    def reshape(self, shape, order='C'):
-        """Gives a new shape to a sparse matrix without changing its data."""
-        raise NotImplementedError
+    def reshape(self, *args, order='C'):
+        """Gives a new shape to a sparse matrix without changing its data.
+        Args:
```

Comment:
You can see the document built from this pull request [here](https://cupy--5301.org.readthedocs.build/en/5301/). It seems you need an additional blank line before `Args:` part; otherwise the description about `shape` is wrongly rendered:

```suggestion

        Args:
```